_posts/2022-10-10-Announcing-Data-Prepper-2.0.0.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title:  "Announcing Data Prepper 2.0.0"
+authors:
+- dlv
+- oeyh
+date: 2022-10-10 15:00:00 -0500
+categories:
+  - technical-post
+---
+
+The Data Prepper maintainers are proud to announce the release of Data Prepper 2.0. This release makes Data Prepper 
+easier to use and helps you improve your observability stack based on feedback from our users. Data Prepper 2.0 retains
+compatibility with all current versions of OpenSearch.
+
+Here are some of the major changes and enhancements made for Data Prepper 2.0.
+
+## Conditional routing
+
+Now Data Prepper 2.0 supports conditional routing to help pipeline authors send different logs to specific OpenSearch clusters.
+
+One common use-case this supports is to reducing the volume of data going to some clusters.
+When you want info logs that produce large volumes of data to go to a cluster or index with more frequent rollovers or 
+deletions to clear out these large volumes of data, you now configure pipelines to route your data.
+
+
+Simply pick a name appropriate for the domain and a Data Prepper expression.  
+Then for any sink that should only have some data coming through, define one or more routes to apply Data Prepper will evaluate 
+these expressions for each event to determine which sinks to route these events to. Any sink that has no routes defined will accept all events.
+
+For example, consider an application log that includes log data. A typical Java application log might look like the following.
+
+```
+2022-10-10T10:10:10,421 [main] INFO org.example.Application - Saving 10 records to SQL table "orders"
+```
+
+The text that reads `INFO` indicates that this is an INFO-level log. Data Prepper pipeline authors can now route logs with this level to only certain OpenSearch clusters.
+
+The following example pipeline takes application logs from the `http` source. This source 
+accepts log data from external sources such as Fluent Bit. 
+
+The pipeline then uses the `grok` processor to split the log line into multiple fields. 
+The `grok` processor adds named `loglevel` to the event. Pipeline authors can use that field in routes. This pipeline has two OpenSearch sinks. The first sink only receives 
```

Comment:
Let's break this up a little more:

---

The pipeline then uses the `grok` processor to split the log line into multiple fields. The `grok` processor adds a named `loglevel` to the event. Pipeline authors can use that field in routes.

This pipeline contains two OpenSearch sinks. The first sink will only receive logs with a log level of `WARN` or `ERROR`. Data Prepper will route all events to the second sink.

Reply:
I took your suggestion and made one clarification by adding "field" which you can see here: "... adds a 
_field_ named `loglevel` ..."